packages/hoppscotch-selfhost-web/src/platform/auth/auth.platform.ts

```diff
@@ -317,9 +318,20 @@ export const def: AuthPlatformDef = {
   async setEmailAddress(_email: string) {
     return
   },
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+
   async setDisplayName(name: string) {
-    return
+    if (!name) return E.left("USER_NAME_CANNOT_BE_EMPTY")
+
+    const res = await updateUserDisplayName(name)
+
+    if (E.isRight(res)) {
+      probableUser$.next({
+        ...probableUser$.value,
+        displayName: res.right.updateDisplayName.displayName,
+      } as HoppUser)
```

Comment:
If we add a check to see if we've a probable user / current user and returns an error in that case, we can avoid this casting.